bin/create_dot_env.sh

```diff
@@ -28,7 +28,7 @@ shift "$((OPTIND-1))";
 
 # Defaults
 SECURE_COOKIE_SECRET="chocolate chip"
-MUPPET_CHARACTER="rizzo"
+MUPPET_CHARACTER="pepethekingprawn"
```

Comment:
We'll need to add a picture of this character to the site somewhere.  Maybe it could be an easter egg.